docs/UserGuide.md

```diff
@@ -266,69 +262,95 @@ Example:
 
 ### 6.7. Undoing your previous command:
 
-Restores the WorkBook to the state before the previous <em>undoable</em> command was executed
+If you executed a wrong <em>undoable</em> command or changed your mind after executing an <em>undoable</em> command, this command 
+will restore the WorkBook to its earlier state before the previous <em>undoable</em> command was executed.
```

Comment:
How to tell if my command is undo-able? maybe an example to elaborate will be useful!

And also kind of tough to digest, maybe something simpler like:
I accidentally_ cleared my list of internship applications, what do I do??
WorkBook helps undo/revert your changes if you happen to change your mind after performing an undesirable action ......